### DISCUSSION TITLE
Correct motherboard fan settings

### DISCUSSION THREAD

**Discussion by funwithfans** [Rem0o/FanControl.Releases#2565] [General]
Hi,

I'm about to start using fancontrol for the first time and trying to understand what the correct motherboard settings should be.

System:

- Asrock B650 PG Lightning

- 6 x Thermaltake Swafan EX12 RGB through proprietary controller
   - controller connected to board via internal USB 2.0

- 2 x 160mm Lian Li RGB case fans included with Lian Li 216 RGB, run through included case RGB hub
   - connected to board via Lian Li hub which plugs into CHA_FAN4/WP
   - CHA_FAN4/WP - in BIOS, "control mode" is "AUTO" (options: AUTO/DC/PWM)
                             - in BIOS, "setting" is "SILENT"
                             - in BIOS, "temp source" is "monitor M/B"

- Arctic Liquid Freezer 2 360 A-RGB rev. 3
   - pump connected to board > CPU_FAN1
   - CPU_FAN1 - in BIOS, "control mode" is not present
                     - in BIOS, "setting" is "SILENT"
                     - in BIOS, "temp source" is not present

I've searched similar questions and FAQ so have read that any kind of "smart" setting is to be avoided.

Questions:

- should "AUTO" as a "control mode" be interpreted as "SMART"?
- should CHA_FAN4/WP (2x Lian Li 160mm RGB fans) be left as "AUTO" or changed to "PWM"?
- do I definitely need the Thermaltake plugin? If so, on the page for the plugin, how do I download and/or use it?

This is a first build and I'm inexperienced with this level of tinkering so please feel free to speak like I'm 5 years old. Thanks for reading and for any help offered.

**Comment by Rem0o** [comment]
1 Auto == smart yes
2 PWM
3 If you want to be able to control the fans on the controller, yes. Download the dll and drag/drop it in the plugins folder.

**Reply by funwithfans** [reply]
Thanks a lot, on the dll page, am I supposed to "download raw file" from the bar that begins with "code/blame"? Sorry about my inane questions!

**Reply by Rem0o** [reply]
https://github.com/fu-raz/FanControlThermaltake/releases/tag/v1.0.0

**Reply by funwithfans** [reply]
I was in the wrong place entirely lol, thanks very much for your help.

**Reply by funwithfans** [reply]
I've not yet run the initial setup that detects fans etc... should the plugin be applied before or after the initial detection or does it not matter?

**Reply by Rem0o** [reply]
You can always rerun the setup process on top of whatever you had.
